### PR TITLE
Add edgarlr/magazine starter

### PR DESCRIPTION
**Pull request by edgarlr**
Add magazine starter kit to starters.yml

### CODE REVIEW

File: starters/starters.yml [hunk 1]
```diff
@@ -225,3 +225,33 @@
...
+    - PWA Optimized (installable)
+    - Preview unpublished content.
+    - Seaarch module and hooks.
+    - Static Site Generated with Next.js
+    - Dynamically generated sitemap
+    - Content creation and managment from Strapi CMS (No code necessary).
+    - Google Analytics
+  youtube:
+  how_to_install: Clone the repository and follow the readme instructions.
+  command:
```
Comment (markkaylor) [inline starters/starters.yml:256]: You should add the starter CLI command here

**Review by markkaylor — CHANGES_REQUESTED** [review]
Hey @edgarlr 

Thanks so much for submitting your starter and template.  

Could you update your starter to be inside a `starter` directory, and then add a `starter.json` file at the root which points to the template github repo:

```bash
{
  "template": "https://github.com/edgarlr/strapi-template-magazine"
}
```

Take a look at this starter as an example: [https://github.com/strapi/strapi-starter-next-corporate](https://github.com/strapi/strapi-starter-next-corporate)

That way it can be used with the starter CLI:

```bash
npx create-strapi-starter my-starter https://github.com/edgarlr/magazine
```

You can learn more about the starter CLI here: [https://strapi.io/blog/announcing-the-strapi-starter-cli](https://strapi.io/blog/announcing-the-strapi-starter-cli)

**Comment by edgarlr** [comment]
@markkaylor Sure, I'll update this!

**Comment by edgarlr** [comment]
Hey @markkaylor, I updated the PR, if you want to take a look at it again!

File: starters/starters.yml [hunk 2]
```diff
@@ -225,3 +225,34 @@
...
+    yarn: yarn create strapi-starter my-site https://github.com/edgarlr/strapi-starter-magazine
+    npm: npx create-strapi-starter my-site https://github.com/edgarlr/strapi-starter-magazine
```
Comment (markkaylor) [inline starters/starters.yml:258]: If you change the name of your repo to `strapi-starter-next-magazine` you would be able to use the shortcut here too: `npx create-strapi-starter edgarlr/next-magazine`.

More info at the bottom of this article: https://strapi.io/blog/announcing-the-strapi-starter-cli

**Review by markkaylor — CHANGES_REQUESTED** [review]
Looks good, if you don't mind changing the repo and using the shortcut that would be awesome.  Once you have some seed data on the template we can merge.

File: starters/starters.yml (hunk 2 above)
Comment (edgarlr) [reply starters/starters.yml]: @markkaylor done! I updated the name of the repository and the commands, and also I already added the seed data on the template!

**Comment by edgarlr** [comment]
Hey @markkaylor, can you please review this PR again? I already addressed the changes and added the seed data to the template 🙂

**Comment by markkaylor** [comment]
Hi @edgarlr I think it is good to go, I'll take another look later today to be sure.   It might be a week or two before we can merge.  I'll keep you posted.

**Comment by edgarlr** [comment]
@markkaylor  Sounds great, thanks! If you could take a look at the other PR as well, that would be great! I'll stay tuned!

**Pull request closed without merging**